src/index.js

```diff
@@ -41,6 +45,7 @@ export default class Server {
 
                 imagefile.mv( './resources/upload/' + imagefile.name, () => {
                     console.log('File uploaded');
+                    res.sendFile(path.join(__dirname, '../resources/pages/SuccessUpload.html'));
```

Comment:
Can you see `Upload file!` text on the page after uploading is finished?
Seems like we should remove `res.end();`. Please check it.
We can even add an assertion in our `uploading` test for this text.